public/app/core/angular_wrappers.ts

```diff
@@ -102,4 +103,10 @@ export function registerAngularDirectives() {
   ]);
 
   react2AngularDirective('reactProfileWrapper', ReactProfileWrapper, []);
+
+  react2AngularDirective('lokiAnnotationsQueryEditor', LokiAnnotationQueryEditor, [
```

Comment:
Don't like this is defined here but we already have plugins parts defined here as directives for react - angular interop so I figured it makes more sense to keep the consistency.

Reply:
I think it is fine for now and we can remove it when we allow any DataFrame response as annotation.